<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrategyDlg</class>
 <widget class="QDialog" name="StrategyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Strategy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="modeLabel">
             <property name="text">
              <string>&amp;Mode:</string>
             </property>
             <property name="buddy">
              <cstring>modeComboBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="modeComboBox">
             <item>
              <property name="text">
               <string>INIT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>STEP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FINAL</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>POSTRUN</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="applyPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>A&amp;pply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Re&amp;set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="postFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="postLabel">
          <property name="text">
           <string>&amp;Label:</string>
          </property>
          <property name="buddy">
           <cstring>postLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="postLineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="triggerFrame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="triggerLabel">
          <property name="text">
           <string>&amp;Trigger:</string>
          </property>
          <property name="buddy">
           <cstring>triggerLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="triggerLineEdit"/>
        </item>
       </layout>
       <zorder>triggerLabel</zorder>
       <zorder>triggerLineEdit</zorder>
       <zorder>triggerLineEdit</zorder>
       <zorder></zorder>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="docLabel">
       <property name="text">
        <string>&amp;Doc:</string>
       </property>
       <property name="buddy">
        <cstring>docTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="docTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>modeComboBox</tabstop>
  <tabstop>postLineEdit</tabstop>
  <tabstop>triggerLineEdit</tabstop>
  <tabstop>docTextEdit</tabstop>
  <tabstop>applyPushButton</tabstop>
  <tabstop>resetPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
